Library/release.txt

```diff
@@ -0,0 +1 @@
+2.2
```

Comment:
This file should not be checked in. It is for the deployed library, not the git repo.

Reply:
I think I may have copied the Master Library folder at some point instead of pulling from the repo. Should be fixed now.